dev-env/bin/dade-copyright-headers

```diff
@@ -209,7 +209,7 @@ def process_file(filename, check_only):
            # copyright header existed, but was out of date, replace it.
           fp.write("".join(before_header))
           fp.write(header)
-          fp.write("".join(after_header))
+          fp.write("".join(after_header[:-1]))
```

Comment:
What happens if there is no trailing newline in the file? Will we discard the last line in this case?

Reply:
Thanks for the explanation!